cmd/kola/options.go

```diff
@@ -169,6 +171,11 @@ func syncOptionsImpl(useCosa bool) error {
 		return err
 	}
 
+	kola.CosaBuild, err = cosa.ParseBuild(kola.Options.CosaBuild)
```

Comment:
Shouldn't we check first here if `kola.Options.CosaBuild` is not empty?

Reply:
Fixed, thanks!  (Imagine if golang had had `Option<T>`...)